src/plugins/viewDatumAction/ViewDatumAction.js

```diff
@@ -21,7 +21,7 @@
  *****************************************************************************/
 
 import MetadataListView from './components/MetadataList.vue';
-import Vue from 'vue';
+import mount from 'utils/mount';
 
 export default class ViewDatumAction {
   constructor(openmct) {
```

Comment:
Can we chance _openmct to just openmct for consistency if we don't have a reason for the underscore?

Reply:
ya

Reply:
thanks!